docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -49,7 +49,13 @@ Some other important facts about events:
 - Events are not stored on-chain. Only events from the most recent couple of transactions are retained by consensus nodes. After that, they will be discarded, but are still accessible on archive nodes. In the near future, we plan to add an API to easily fetch events from an archive node.
 - You can't refer to previously emitted events in a smart contract, because there is no way of proving that the events you refer to are actually the events emitted by that contract.
 
-This is all you need to know about events! Think of them as a convenience feature -- a light-weight way of attaching information about your smart contract execution, which would otherwise get lost. Don't treat them as fully-fledged storage which can be safely accessed in smart contracts.
+:::tip
+
+A zkApp can retrieve events and actions from one or more Mina archive nodes. If your smart contract needs to fetch events and actions from an archive node, see [How to Fetch Events and Actions](fetch-events-and-actions).
+
+:::
+
+This is all you need to know about events! Think of them as a convenience feature -- a light-weight way of attaching information about your smart contract execution, which would otherwise get lost. Don't treat them as fully-fledged storage that can be safely accessed in smart contracts.
```

Comment:
should we use an actual em dash here? `—`

Reply:
Suggestion. 'Getting lost' seems negative and I wonder if this suggestion gets to the intent better:

`, which would otherwise get lost`   >  `, to expose the event to the outside world such as your UI`